x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -224,6 +224,7 @@ const getSavedObjectTypes = (
         output_id: { type: 'keyword' },
         package: {
           properties: {
+            id: { type: 'keyword' },
```

Comment:
Does touching these saved object mappings require anything else? Is this what migrations are for? We don't necessarily have to persist this ID, and we could instead just implement a workaround to allow it at the request/response level.